tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,8 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert result == False
```

Comment:
I suggest adding a check to verify that the inventory wasn't changed by the remove action: 
```suggestion
    assert result == False
    assert len(vendor.inventory) == 3
```